documentation-site/pages/components/link.mdx

```diff
@@ -5,6 +5,15 @@ This source code is licensed under the MIT license found in the
 LICENSE file in the root directory of this source tree.
 -->
 
+# When to use the default version:
```

Comment:
Could you please position this description above the `API` heading below on line 30? 

If you can, please write the description as if the reader is not aware of the option. For example, this could be described as: "Your application may need to draw more attention to some links on the page. Apply the `animateUnderline` prop to the component if the link is a call-to-action or initiates a UX flow." 

Also, please consider that many visitors to this website will not be uber employees. Expressing brand personality is not something that audience is interested in and is often a detractor towards using baseui in their personal projects. It's best to leave those points out of public docs